FightPandemics/Features/Post/CreatePost/OfferRequest/OfferRequest.swift

```diff
@@ -0,0 +1,135 @@
+        offerBtn.setTitleColor(UIColor.white, for: .selected)
+        offerBtn.frame.size.height = 45
+        offerBtn.frame.size.width = UIScreen.main.bounds.width - 46
+        offerBtn.backgroundColor = UIColor(hexString: "#F2F2F2")
+        offerBtn.clipsToBounds = true
+        offerBtn.layer.masksToBounds = true
+        offerBtn.layer.cornerRadius = 4
+        self.addSubview(offerBtn)
+        reqBtn.setAttributedTitle(NSAttributedString(string: "OfferRequestRequestBtn".localized, attributes: [NSAttributedString.Key.font: Fonts.dmSansRegular.customFont(size: 12), NSAttributedString.Key.foregroundColor: UIColor.black]), for: .normal)
+        reqBtn.setTitleColor(UIColor.white, for: .selected)
+        reqBtn.frame.size.height = 45
+        reqBtn.frame.size.width = UIScreen.main.bounds.width - 46
+        reqBtn.backgroundColor = UIColor(hexString: "#F2F2F2")
+        reqBtn.clipsToBounds = true
+        reqBtn.layer.masksToBounds = true
+        reqBtn.layer.cornerRadius = 4
+        self.addSubview(reqBtn)
+        closeBtn.setAttributedTitle(NSAttributedString(string: "OfferRequestCloseBtn".localized, attributes: [NSAttributedString.Key.font: Fonts.poppinsRegular.customFont(size: 15), NSAttributedString.Key.foregroundColor: UIColor(hexString: "#425AF2")]), for: .normal)
+        closeBtn.frame.size.width = ((UIScreen.main.bounds.width - 63) / 2)
+        closeBtn.frame.size.height = 45
+        closeBtn.clipsToBounds = true
+        closeBtn.backgroundColor = UIColor.clear
+        closeBtn.layer.masksToBounds = true
+        closeBtn.layer.cornerRadius = closeBtn.frame.size.height / 2
+        closeBtn.layer.borderColor = UIColor(hexString: "#425AF2").cgColor
+        closeBtn.layer.borderWidth = 1
+        self.addSubview(closeBtn)
+        confirmBtn.setAttributedTitle(NSAttributedString(string: "OfferRequestConfirmBtn".localized, attributes: [NSAttributedString.Key.font: Fonts.poppinsRegular.customFont(size: 15), NSAttributedString.Key.foregroundColor: UIColor(hexString: "#425AF2")]), for: .normal)
+        confirmBtn.frame.size.width = ((UIScreen.main.bounds.width - 63) / 2)
+        confirmBtn.frame.size.height = 45
+        confirmBtn.clipsToBounds = true
+        confirmBtn.backgroundColor = UIColor.clear
+        confirmBtn.layer.masksToBounds = true
+        confirmBtn.layer.cornerRadius = confirmBtn.frame.size.height / 2
+        confirmBtn.layer.borderColor = UIColor(hexString: "#425AF2").cgColor
+        confirmBtn.layer.borderWidth = 1
+        self.addSubview(confirmBtn)
+    }
+    func setUpConstraints () {
+        self.translatesAutoresizingMaskIntoConstraints = false
+        topLine.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint(item: topLine, attribute: .centerX, relatedBy: .equal, toItem: self, attribute: .centerX, multiplier: 1, constant: 23).isActive = true
```

Comment:
For the centerX constraint, we actually want it at the center, hence, the constant value should be **0** and not **23**

Reply:
I actually gave a 0 but it wasn't drawn at the center, I though the constant is going to give margin

Reply:
I tried and you were right, thanks again